regparser/tree/depth/markers.py

```diff
@@ -6,15 +6,31 @@
 from regparser.utils import roman_nums
 
 
+def emphasize(marker):
+    """The final depth levels for regulation text are emphasized, so we keep
+    their <E> tags to distinguish them from previous levels. This function
+    will wrap a marker in an <E> tag"""
+    marker_plain = deemphasize(marker)
+    return '<E T="03">{}</E>'.format(marker_plain)
```

Comment:
GPO uses an XSD and has a half-completed manual documentation of their schema, but no, not really.